extensions-core/kafka-indexing-service/src/test/java/io/druid/indexing/kafka/KafkaIndexTaskTest.java

```diff
@@ -1491,7 +1491,7 @@ private void makeToolboxFactory() throws IOException
         derby.metadataTablesConfigSupplier().get(),
         derbyConnector
     );
-    taskLockbox = new TaskLockbox(taskStorage);
+    taskLockbox = new TaskLockbox(taskStorage, 300000);
```

Comment:
I think the timeout of 300 seconds is too long. Of course, waiting for 300 seconds is not usual, but if the lock is acquired after waiting for 300 seconds, it is likely to make unit tests failed eventually on travis due to the job timeout. I think it would be better to make tests failed earlier rather than waiting for such a long time. This is same for other tests.